projects/commudle-admin/src/app/feature-modules/events/components/collaborating-communities/collaborating-communities.component.scss

```diff
@@ -32,3 +32,20 @@
     }
   }
 }
+
+::ng-deep .cdk-overlay-container {
```

Comment:
Can we change the styles of the logo in the search results without using `.cdk-overlay-container`? Asking because these styles might affect some other component somewhere else